geoportal/c2cgeoportal_geoportal/lib/lingua_extractor.py

```diff
@@ -422,7 +422,7 @@ def _import_layer_wms(self, layer, messages):
         if layer.geo_table is not None and layer.geo_table != "":
             try:
                 from c2cgeoportal_geoportal.views.layers import get_layer_class
```

Comment:
Seems wreid to import from views here. Method get_layer_class might be in lib.db_reflexion.

Reply:
It can't be it make initialization issue ...